packages/ffe-accordion/less/ffe-accordion.less

```diff
@@ -12,7 +12,7 @@
     border: 2px solid transparent;
     background-color: @ffe-farge-hvit;
 
-    &__heading {
+    &__heading.ffe-h6 {
```

Comment:
Litt usikker på om dere ønsker det slik eller at jeg legger på en klasse til på [`<Heading6>`](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-accordion-react/src/AccordionItem.js#L52) elementet, eller andre måter å fiske spesifisitet på

Reply:
Litt utenfor dette, men litt usikker på om det blir riktig at vi sender med headingLevel,og setter det som aria-level. Føler det riktige her blir å sette <h> taggen som det samme som headingLevel og heller sende med &__heading klassen som extender ffe-h6. 

ref Aria-level: https://uxplanet.org/should-i-use-aria-level-in-the-h-tag-8efc2cd0a114

Reply:
Det aria-level tingen har jag fått fra denne boken før den intresserte. 
http://book.inclusive-components.design/

Reply:
Mulig aria-level anbefalingene har endret seg litt over tid. Den boka du refererer til ser ut til å være fra 2017, mens artikkelen jeg linket til er fra 2020. Tenker uansett at det virker som litt bad practice å bruke feil h-tag, og at det blir mer riktig å overskrive stylingen, men bevare riktig h-tag. :)

Reply:
Ser ut som den blir tolket riktigt iaf. :)
![image](https://user-images.githubusercontent.com/2248579/195086494-54002a04-d1db-4145-a252-ede8c07a44f9.png)

Reply:
Jag er i utgangspunkt ikke oenig i vanliga fall i det att det er best og bruka riktig element. Jag liker denne fordi du slipper veta om klassen ffe-h6 + att du kan bruke jsx.

Reply:
Føler det skjer endringer på best practice hele tiden 😅 Så kjempe fint at du delte historikken bak det og at man kan ha en diskusjon på det! :)